simul/platform/mininet.go

```diff
@@ -347,7 +346,10 @@ func (m *MiniNet) parseServers() error {
 		if len(h) > 0 {
 			ips, err := net.LookupIP(h)
 			if err != nil {
-				return err
+				if err2 := CheckOutOfFileDescriptors(); err2 != nil {
```

Comment:
nitpick: I don't think you need `err2`. What's defined inside the `if` statement won't affect the outer scope.

Reply:
yes, but I think it's not readable if you use the same. IIRC, this will even be an error with the linter where he'll say: "shadowing outer variable" or such.